GreysonElkins/exquisite-corpse
# Feature/nh prompt display

#### What's this PR do?
- Starts setting up ApiHelper
- Sets up writing prompt display to follow user stories (it can display one randomly selected from a given genre, all genres, or none at all)
- Adds some testing coverage for conditional renders
#### Where should the reviewer start?
ApiHelper.js, then StorySetup.js - check out handleSubmit
#### How should this be manually tested?
Click new story, select a prompt and see one show up! If the prompt option isn't selected, none should display.
#### What are the relevant tickets?
Resolves #31, resolves #32 
#### Screenshots (if appropriate)
![Screen Shot 2020-09-13 at 11 25 50 PM](https://user-images.githubusercontent.com/60753853/93043800-82a94200-f618-11ea-9ce8-3088fce1f1d9.png)
#### Questions:
- At this point I'm realizing the StorySetupView isn't doing too much, whereas StorySetup is doing a lot of the heavy lifting - we could refactor to have the former hold on to a few more methods/info, but everything's working as is. How could we leverage StorySetupView more? Lemme know what y'all think!
- Gonna make an issue on the BE repo, but there's a few prompt endpoints that come up with an empty response every once in a while.
